src/olympia/addons/tests/test_serializers.py

```diff
@@ -536,6 +537,19 @@ def test_no_license(self):
         assert result['id'] == self.version.pk
         assert result['license'] is None
 
+    def test_file_webext_permissions(self):
```

Comment:
At the moment only the tests in `AddonSerializerOutputTestMixin` are tested against both database and elasticsearch, and as a result the test you're adding here will not be tested with ES data, since it's in `TestVersionSerializerOutput`.

My suggestion (but feel free to refactor the tests differently to achieve similar or better coverage) : keep that test, but also add permissions in the basic addon created in `AddonSerializerOutputTestMixin.test_basic`, and modify `AddonSerializerOutputTestMixin._test_version` to also test permissions.